tests/ColorRampDropdown.test.tsx

```diff
@@ -40,43 +40,6 @@ describe("ColorRampDropdown", () => {
       expect(element).toBeInTheDocument();
     });
 
-    it("calls the onChange callback when a selection is made", () => {
-      const callback = (_key: string): void => {
-        // do nothing
-      };
-      const mockCallback = vi.fn(callback);
-
-      render(
-        <ColorRampDropdown
-          selectedRamp={"map1"}
-          colorRampsToDisplay={["map1", "map2", "map3"]}
-          categoricalPalettesToDisplay={[]}
-          knownColorRamps={customColorRamps}
-          onChangeRamp={mockCallback}
-          useCategoricalPalettes={false}
-          numCategories={0}
-          selectedPalette={defaultPalette}
-          onChangePalette={(): void => {}}
-        />
-      );
-      const elements = screen.getAllByRole("button");
-
-      // Expect maps to be ordered according to the color ramp, and skip the first button which is the main selector.
-      for (let i = 1; i < elements.length - 1; i++) {
-        // Ignore the last button, which is for reversing
-        const buttonElement = elements[i];
-        fireEvent.click(buttonElement);
-      }
-
-      // Should be called three times, in order, one time for each element.
-      expect(mockCallback.mock.calls.length).to.equal(3);
-      expect(mockCallback.mock.calls).deep.equals([
-        ["map1", false],
-        ["map2", false],
-        ["map3", false],
-      ]);
-    });
-
```

Comment:
I tried fixing this test using a plugin (https://www.npmjs.com/package/react-select-event) but couldn't quite get it working. I'm comfortable removing it though now that the selection behavior is being handled by a battle-tested library.